.gitignore

```diff
@@ -1,73 +1,6 @@
-### Windows ###
```

Comment:
I liked how this was organized so we knew why specific things were being excluded.  We should bring it back if possible.